src/Styleguide/Components/MarketInsights.tsx

```diff
@@ -1,19 +1,11 @@
 import React from "react"
 import { Responsive } from "../Utils/Responsive"
-import { BorderBox } from "../Elements/Box"
+import { Box, BorderBox } from "../Elements/Box"
+import { Flex } from "../Elements/Flex"
 import { Sans } from "@artsy/palette"
-import { themeGet } from "styled-system"
-import styled from "styled-components"
 
-const TextWrap = styled.div`
-  display: block;
-
-  @media ${themeGet("mediaQueries.xs")} {
-    display: flex;
-    flex-direction: column;
-    margin-bottom: 10px;
-  }
-`
+const wrapper = xs => props =>
+  xs ? <Flex flexDirection="column" mb={3} {...props} /> : <Box {...props} />
```

Comment:
One thing I’ve started wondering with the approach of rendering different components at different breakpoints is if this is working against React’s reconciliation (i.e. being unable to re-use existing DOM elements/iOS views but adjusting them)

Reply:
Well, on web both of these things result in `div` elements. The classes and attributes are different so there would be some updates, but the reconciliation process would work the same. On native, this flag wouldn't change so the returned result should always be the same.